components/header.js

```diff
@@ -1,20 +1,29 @@
+import { Trans, useTranslation } from "react-i18next";
 import { Navigation } from "~/components/navigation";
 
 export function Header() {
+  const { t } = useTranslation();
+
   return (
-    <header className="absolute left-0 z-20 flex w-full items-center p-10 text-gray md:text-white">
+    <header className="absolute left-0 z-20 flex w-full items-center gap-8 p-10 text-gray md:text-white">
       <Navigation />
 
-      <span className="ml-8 hidden font-light md:block">
-        Illustrations by{" "}
-        <a
-          className="font-normal underline"
-          href="https://katerodman.com/"
-          target="_blank"
-          rel="noreferrer"
+      <span className="hidden font-light md:block">
+        <Trans
+          t={t}
+          i18nKey="header.credits"
+          className="hidden font-light md:block"
         >
-          Kate Rodman
-        </a>
+          Illustrations by{" "}
+          <a
+            className="font-normal underline"
+            href="https://katerodman.com/"
+            target="_blank"
+            rel="noreferrer"
+          >
+            Kate Rodman
```

Comment:
I just noticed that `Kate Rodman` is both in the translation files and here. Shouldn't the translations have a placeholder for the name?

I think the translation would look something like
```json
  "header.credits": "Illustrations by <2>{{name}}</2>",
```
```tsx
const name = "Kate Rodman"
<Trans
  t={t}
  i18nKey="header.credits"
  className="hidden font-light md:block"
>
  Illustrations by{" "}
  <a
    className="font-normal underline"
    href="https://katerodman.com/"
    target="_blank"
    rel="noreferrer"
  >
    {{name}}
  </a>
</Trans
```

I'm kinda confused by the syntax but what I'm trying to say is that the name should appear only once :)

Reply:
The syntax is weird for sure. It would surely be better to pass a `name` variable to the translation, yea. Although I don't think Kate is going to change her name anytime soon 🙃